allen-cell-animated/nucmorph-colorizer
# feat: Loading feature thresholds from URL

Problem
=======
Closes #124!

Solution
========
- Feature thresholds and current color ramp min+max are now saved to the URL if edited from the defaults.
- The app will now load the color ramp and thresholds from the URL on startup!
- Also refactors tests and other URL handling code to reduce assumptions about formatting in the utility scripts.

## Type of change
* New feature (non-breaking change which adds functionality)

Steps to Verify:
----------------
1. Open the preview link.
2. Set the feature color map range. You should see the URL update.
3. Set feature thresholds in the filters pane. You should also see the URL update.
4. Copy the URL and paste it into a new tab or browser window.
5. The thresholds + color map range should persist.
